kotlin/internal/jvm/android.bzl

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+load(
+    ":jvm.bzl",
+    _kt_jvm_library = "kt_jvm_library",
+)
+
+def _kt_android_artefact(name, srcs = [], deps = [], plugins = [], **kwargs):
```

Comment:
Bazel uses `artifact` everywhere, let's standardize on that?

Reply:
updated in new PR.